_config.yml

```diff
@@ -1,3 +1,3 @@
 name: TestAnything.org
 markdown: redcarpet
-pygments: true
+highlighter: pygments
```

Comment:
It's a syntax change in jekyll. I only know this because I got a deprecation warning when I ran `jekyll serve`.